src/cmd-import

```diff
@@ -114,6 +117,38 @@ def generate_lockfile(tmpd, ostree_commit):
     return tmpf
 
 
+def generate_commitmeta(tmpd, ostree_commit, build_meta):
+    tmpf = os.path.join(tmpd, 'commitmeta.json')
+    out = subprocess.check_output(['rpm-ostree', 'db', 'list', '--repo', 'tmp/repo', ostree_commit], encoding='utf-8')
+    ostree_linux = ''
+    rpmdb = []
+    for line in out.splitlines():
+        if not line.startswith(' '):
+            continue
+
+        # Do we have epoch in package names?
+        n, v, ra = line.strip().rsplit('-', 2)
+        r, a = ra.rsplit('.', 1)
+        rpmdb.append([n, "0", v, r, a])
```

Comment:
I think we need the `epoch`, see https://github.com/openshift/os/blob/master/scripts/generate-metadata#L39

Reply:
@HuijingHei 
The rpm package name from the command used in this PR doesn't have epoch. Also, on the commitmeta.json created with the traditional way of cosa build I have noticed epoch as 0. Do we have any other way to get epoch?